istresearch/mailroom
# P4-1531 - Initial repo setup with CI, PR template, and allow custom schemes

# For the Reviewer

When this is complete, you should approve the PR via github.

## Summary
Setup CircleCI, the PR template, and added an option to allow custom URN schemes

#### Release Note
Added a configuration to mailroom to let custom URN schemes pass validation

#### Breaking Changes
None

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

- Setup an engage 4.0.2 stack using the `istresearch/mailroom:ci-4.0.2-dev-P4-1531` mailroom image
- Add the postmaster channel scheme to mailroom, using the `MAILROOM_CUSTOM_SCHEMES` env var
- Add a postmaster channel, and ensure you can send messages to it.
